packages/www/pages/docs/guides/debugging-guide.mdx

```diff
@@ -0,0 +1,71 @@
+
+This step-by-step guide will help you debug common stream playback problems. Try these steps before reaching out to Livepeer.com directly.
+
+We will use the [Livepeer.com test player](/app/test-player) to debug. You must be logged into the Livepeer.com dashboard. 
+
+#### Prerequisite: Locate your playback URL. 
+
+Your playback URL is specific to the stream ingest region. 
+- This [Livepeer.com API guide](/docs/guides/api/playback-a-live-stream) touches on finding your playback URL by combining a base url with a `playbackId`. If you don't know your stream `playbackId`, you can retrieve it via [`GET /stream/<id>`](/docs/api/stream/get-stream).
+- You can also find the playback URL on the Livepeer.com dashboard stream page, `https://livepeer.com/app/stream/<stream-id>`. Learn more [here](/docs/guides/dashboard/playback-a-stream).
+
+## Step 1
+
+Load your playback URL in the [Livepeer.com test player](/app/test-player). If you get a "Stream not found" error, confirm that your URl is correct. It should look something like this:  https://region-cdn.livepeer.com/hls/123456abcdef7890/index.m3u8
+
+## Step 2
+
+Does anything playback in *your* own player?
+
+- **No:** Go to the next step.
+- **Yes, but the quality is poor:** Go to step 5.
+
+## Step 3
+
+Does anything playback in the [Livepeer.com test players](/app/test-player)?
+
+- **Yes, the quality is great:** There may be something wrong with your player configuration.
+- **No:** Go to the next step.
+- **Yes, but the quality is poor:** Go to step 5
+
+## Step 4
+
+Check the stream status below the Livepeer.com test player. Is it active?
+
+- **No:** No video is being ingested. Confirm your or your streamer’s broadcast software RTMP configuration is set up properly. 
+- **Yes:** Go to the next step.
+
+## Step 5
+
+Check if you are transcoding. Is there only one `profile` or zero `profiles` listed next to `Playback settings` below the Livepeer.com test player? 
+
+- **No:** You are not transcoding your source stream, and only the source profile is available for playback (no ABR) via the Livepeer.com CDN. There may be something wrong with your source stream. Confirm the keyframe interval is set to 2 in your broadcasting software. Try switching the broadcast software internet provider.
+<br />Or, depending on the resolution, fps and bitrate of your source and the bandwidth of your internet, playback may be slow and choppy. Consider transcoding your ingested video to take advantage of ABR in playback.
+- **Yes:** If two or more `profiles` are listed. Go to the next step.
+
+## Step 6
+
+Do all the `profiles` have the same FPS? Between 5 FPS and 60 FPS work best. Setting FPS to `0` is also acceptable and passes through the source FPS to the transcoded renditions.
+
+- **No:** Your `profiles` FPS is configured incorrectly. Set all `profiles` FPS to the same value. 
+- **Yes:** Go to the next step.
+
+## Step 7
+
+Check if something is wrong with the source stream. 
```

Comment:
I'd add ... "The source stream is the stream coming from the broadcasting software you're using to send a video into Livepeer. To do that, read below."